aya/src/bpf.rs

```diff
@@ -315,6 +317,27 @@ impl<'a> BpfLoader<'a> {
         self
     }
 
+    /// Set the max_entries for specified map
```

Comment:
```suggestion
    /// Set the max_entries for specified map.
```

Reply:
Thank you for the review!

I have fixed the docs. 
And I also rebased because #394 was merged.